src/main/java/com/limechain/trie/structure/database/InsertTrieBuilder.java

```diff
@@ -11,18 +11,18 @@
 import com.limechain.trie.structure.nibble.Nibbles;
 import com.limechain.trie.structure.node.InsertTrieNode;
 import com.limechain.utils.HashUtils;
+import lombok.Getter;
 import org.jetbrains.annotations.NotNull;
 
-import java.util.Collections;
 import java.util.List;
 import java.util.Map;
-import java.util.Optional;
 import java.util.stream.IntStream;
 
 public class InsertTrieBuilder {
```

Comment:
Leave a big todo above this class - it should be decomposed into two separate classes, but this is soon to come in a major fix alongside the entire storage host api additions.